templates/devstack/vlan-range-keep-alive.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
This seems unnecessary, we should probably drop it. We're already storing the instance uuid so it's easy for us to tell who's using the range. As long as the instance exists, that range is considered in-use.